src/todoListItem.component.js

```diff
@@ -0,0 +1,21 @@
+import {
+    INPUT_LIST,
+    INPUT_LIST_ITEM,
+
+} from './const.js'
+
+
+
+const templateItem = `<div class="input-list-item">
+                        <div class="input-list-text"></div>
+                        <button class="input-list-btn-remove">Remove</button>
+                        </div>`;
+const templateList = `<div class="input-list"></div>`
```

Comment:
Create separate file 'src/todoListI.component.js' for 
```
const templateList = `<div class="input-list"></div>`

export function getTodoList() {
    return new DOMParser()
        .parseFromString(templateList, "text/html").querySelector(INPUT_LIST);
} 
```

Reply:
Done